CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Tide Contributing Guide
+
+Before submitting your contribution, please make sure to take a moment and read through the following guidelines.
+
+## Issue Reporting Guidelines
+
+- The Issue list of this repo is **exclusively** for bug reports and feature requests.
+- Try to search for your issue, it may have already been answered or even fixed in the `develop` branch.
+- Check if the issue is reproducible with the latest stable version. If you are using a pre-release, please indicate the specific version you are using.
+- It is **required** that you clearly describe the steps necessary to reproduce the issue you are running into. Issues without clear reproducible steps will be closed immediately.
+- If your issue is resolved but still open, don't hesitate to close it. In case you found a solution by yourself, it could be helpful to explain how you fixed it.
+
+## Pull Request Guidelines
+
+- Checkout a topic branch from `develop` and merge back against `develop`.
+    - If you are not familiar with branching please read [_A successful Git branching model_](http://nvie.com/posts/a-successful-git-branching-model/) before you go any further.
+- Follow the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/coding-standards/).
```

Comment:
This is a Golang codebase, no WordPress code here.

Reply:
@valendesigns any specific coding standards that are being followed then instead of WPCS?